types/browser-sync/index.d.ts

```diff
@@ -1,9 +1,10 @@
-// Type definitions for browser-sync
+// Type definitions for browser-sync 2.26.5
```

Comment:
Thanks for adding a version. Can you please remove `"dt-header": false` from tslint.json so that the header is actually checked? Perhaps `"npm-naming": false` too.

Reply:
@sandersn Thank you for your advice. I fixed version to MAJOR.MINOR.
If I remove `"dt-header": false` and run test, `ERROR: 3:26  dt-header  Error parsing header. Expected: /\<https\:\/\/github\.com\/([a-zA-Z\d\-]+)\>/.` because some authors wrote personal web site.
[dt-header](https://github.com/Microsoft/dtslint/blob/master/docs/dt-header.md#dt-header) says:
> Bad:
> ```// Definitions by: My Name <http://geocities.com/myname>```
> Good:
> ```// Definitions by: My Name <https://github.com/myname>```
> Prefer a GitHub username, not a personal web site.

Reply:
Hmm. I guess leave dt-header: false then? I don't want to erase an author. 
Thanks for making the fixes.